swap_meet/vendor.py

```diff
@@ -1,2 +1,77 @@
 class Vendor:
-    pass
+    def __init__(self,inventory=None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+        # if inventory != None else inventory = []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False    
```

Comment:
We can reverse the logic here and use a guard clause to eliminate the `else` clause:

```suggestion
    def remove(self, item):
        if item not in self.inventory:
            return False
        self.inventory.remove(item)
        return item

```